gridstatus/tests/test_ercot.py

```diff
@@ -86,3 +87,107 @@ def test_ercot_get_fuel_mix():
     df = iso.get_fuel_mix("latest").mix
     assert df.shape[0] >= 0
     assert df.columns.tolist() == cols
+
+
+@pytest.mark.slow
+def test_ercot_get_lmp_today_real_time_15_minutes_zone():
+    iso = gridstatus.Ercot()
+    cols = [
+        "Location",
+        "Time",
+        "Market",
+        "Location Type",
+        "LMP",
+    ]
+    df = iso.get_lmp(
+        date="today",
+        market=Markets.REAL_TIME_15_MIN,
+        location_type="zone",
+    )
+    assert df.shape[0] >= 0
```

Comment:
the validation is so similar across these these tests i'd just create a `check_ercot_spp(df, market_type, location_type)` method that contains all the assertions and then use that in each test

at the top of `tests/test_isos.py` you can see how i used this approach elsewhere